src/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -1,52 +1,36 @@
 
 import React, { ReactElement } from 'react';
-import defaults from 'lodash/defaults';
 import { FieldSet, LegacyForms } from '@grafana/ui';
-import { EditorProps, defaultEditor } from './types';
-import { useChangeOptions, useChangeSecureOptions, useResetSecureOptions } from './hooks';
+import { EditorProps } from './types';
+import { useChangeSecureOptions, useResetSecureOptions } from './hooks';
 
-const { SecretFormField, FormField } = LegacyForms;
+const { SecretFormField } = LegacyForms;
 
 export function ConfigEditor(props: EditorProps): ReactElement {
-  const { jsonData, secureJsonData, secureJsonFields } = props.options;
-  defaults(jsonData, defaultEditor);
+  const { secureJsonData, secureJsonFields } = props.options;
 
-  const onUrlFieldChange = useChangeOptions(props);
   const onPersonalTokenChange = useChangeSecureOptions(props);
   const onResetPersonalToken = useResetSecureOptions(props);
 
   return (
     <>
       <FieldSet label="General Settings">
-      <div className="gf-form-inline">
+        <div className="gf-form-inline">
           <div className="gf-form">
-            <FormField
-              data-testid="input-form-field"
-              label="API"
+            <SecretFormField
+              data-testid="input-form-secret-field"
+              isConfigured={Boolean(secureJsonFields.token)}
+              value={secureJsonData?.token || ''}
+              label="Personal Token"
+              // tooltip="Not used yet"

```

Comment:
YANGNI
```suggestion
```

Reply:
I'm going to upload the adjustment in the next pr related to the dashboard, and about eslintrc it's a react bug that keeps appearing.